ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-commands.sh

```diff
@@ -21,7 +21,8 @@ export KUBECONFIG=/tmp/config
 export KRKN_KUBE_CONFIG=$KUBECONFIG
 export NAMESPACE=$TARGET_NAMESPACE
 export ENABLE_ALERTS=False
-
+telemetry_password=$(cat "/secret/telemetry/telemetry_password")
```

Comment:
Instead of adding a second variable you could do something like this to export the variable 

https://github.com/openshift/release/blob/b26ad5b038e5ea13e38390799a8bc388c79b01a1/ci-operator/step-registry/openshift-qe/write-perfscale-results/openshift-qe-write-perfscale-results-commands.sh#L15-L16

Reply:
Fixed it. Thanks.